vxdev/setup-vxdev-machine.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if uname -a | grep Debian; then
+	export DISTRO="Debian"
+else
+	export DISTRO="Ubuntu"
+fi
+
+# Debian doesn't add /sbin/ to default path, but it's needed for groupadd and other commands.
+# If /sbin/ is already in the path, this won't hurt, and it's just for running this script.
+if [[ $DISTRO == "Debian" ]]; then
+    export PATH=${PATH}:/sbin/
+fi
+
+if ! which plustekctl >/dev/null 2>&1
+then
+    echo "error: plustekctl was not found in PATH=${PATH}" >&2
```

Comment:
This is not a change we need for this PR, but it sure would be nice to automate this instead of hoping the user pulled down plustekctl before running the script.